libobs/util/threading-windows.h

```diff
@@ -17,6 +17,11 @@
 #pragma once
 
 #include <intrin.h>
+#include <string.h>
+
+#if !defined(_M_IX86) && !defined(_M_X64)
+#error Processor not supported
+#endif
```

Comment:
This would break #4087 - what can we do as a future step to reintroduce easy Windows ARM64 compatibility?

Reply:
I took a stab at ARM support, but it's not tested.